frontend/src/Shop/Shop.js

```diff
@@ -93,7 +93,7 @@ class Shop extends React.Component {
   }
 
   handleCheckout (event) {
-    const stripe = Stripe('pk_test_51Hu2LSG8eUBzuEBEesc0XBxkWzW7RyHcK7ckx9DqgHL710Mh3BF0NRyowXw82xTUa8vBVIdamkyzkllOzHepdfPc00MoyUNiR6')
+    const stripe = Stripe(process.env.STRIPE_API_KEY)
```

Comment:
How does this work? Do we create a .env file in `frontend/src`?

Reply:
It should be in `frontend`, same directory as package.json. All it needs is the `STRIPE_API_KEY` field.